File: problems/1.c
#include <stdio.h>
#include <stdlib.h>

int main(void)
{
  int sum = 0;

  for (int i = 1; i < 1000; i++)
  {
    if (i % 3 == 0 || i % 5 == 0)
    {
      sum += i;
    }
  }

  printf("Sum of multiples of 3 or 5 below 1000: %d\n", sum);

  return EXIT_SUCCESS;
}

File: solutions/python/1.py
max_number = 1000
sum = 0

for x in range(0, max_number):
  if x % 3 == 0 or x % 5 == 0:
    sum += x

print("Sum of all multiples of 3 or 5 below " + str(max_number) + ": " + str(sum))

